.env.integration-tests

```diff
@@ -9,7 +9,7 @@ DB_PASS=postgres
 DB_USER=postgres
 DEBUG=True
 DELIVERY_CONSOLE_HOST=
-DS_ISSUE_HOST=https://jira.example.com/browse/
+DS_ISSUE_HOST=https://mozilla-hub.atlassian.net/browse/
```

Comment:
You should just be able to take `DS_ISSUE_HOST` out of both `.env.sample` and `.env.integration-test` shouldn't you?

Reply:
oooh you're right